src/tests/ftest/launch.py

```diff
@@ -475,18 +475,18 @@ def get_log_files(config_yaml, daos_files=None):
     """
     # List of default DAOS files
     if daos_files is None:
-        daos_core_test_dir, daos_core_test_log = os.path.split(
+        daos_core_test_dir = os.path.split(
```

Comment:
The `os.path.slit()` returns a tuple and setting the `daos_core_test_dir` to a tuple will cause problems.